docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,104 @@
+| Dependency Dashboard                      | No                                                                                                                                                                           | Yes                                                                                                      |
+| Grouped updates                           | Yes, create [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups) manually     | Yes, use community-provided groups, or create your own                                                   |
+| Upgrades common monorepo packages at once | No                                                                                                                                                                           | Yes                                                                                                      |
+| Supported platforms                       | GitHub only                                                                                                                                                                  | GitHub, GitLab, Bitbucket, Azure, Gitea, see [full list](https://docs.renovatebot.com/modules/platform/) |
+| Supported languages                       | [List for Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems) | [List for Renovate](https://docs.renovatebot.com/modules/manager/)                                       |
+| Customizability                           | Low                                                                                                                                                                          | High                                                                                                     |
+| Show changelogs                           | Yes                                                                                                                                                                          | Yes                                                                                                      |
+| Compatibility score badges                | 1                                                                                                                                                                            | 4                                                                                                        |
+| Built-in to GitHub                        | Yes                                                                                                                                                                          | No, requires app or self-hosting                                                                         |
```

Comment:
`No, requires app or self-hosting`

We can add a link OR explain what self-hosting means

Also, it is probably worth mentioning that there is paid offering for this